earth_enterprise/rpms/build.gradle

```diff
@@ -262,13 +263,14 @@ task openGeeCommonRpm(type: GeeRpm, dependsOn: openGeeSharedFiles) {
     autoFindProvides = true
     autoFindRequires = true
 
-    requires('opengee-postgis', '1.5.8', GREATER | EQUAL)
     requires('chkconfig')
     requires('shadow-utils')
     requires('perl')
```

Comment:
`perl` is unnecessarily added here. It's already auto-detected:

```bash
rpm -qpR opengee-common-5.2.0-1.el7.x86_64.rpm | grep perl

/bin/perl
/usr/bin/perl
perl >= 1:5
perl(Config)
perl(Exporter)
perl(Fcntl)
perl(File::Basename)
perl(File::Spec)
perl(File::stat)
perl(File::Temp)
perl(FindBin)
perl(Getopt::Long)
perl(Getopt::Std)
perl(IO::Handle)
perl(lib)
perl(strict)
perl(vars)
perl(warnings)
perl(WWW::Curl::Easy)
```

Reply:
(see below)